report_xlsx/report/report_abstract_xlsx.py

```diff
@@ -116,3 +116,15 @@ def get_workbook_options(self):
 
     def generate_xlsx_report(self, workbook, data, objs):
         raise NotImplementedError()
+
+    @api.model
+    def _get_new_workbook(self, file_data):
```

Comment:
Better to add these hooks in a separate PR. Anyway, not sure this is the best option, as you are assuming the rest of the manipulation is handling the proper `workbook` object, which may not be. Why not directly hook `generate_xlsx_report` and if you have a boilerplate, process it with the other library?

Reply:
OK, got it.